docs/bots-guide.md

```diff
@@ -133,6 +133,67 @@ You need:
   the `helloworld` bot, you should expect the bot to respond with
   "beep boop".
 
+## Flask Server
+It is a simple python flask server which can be used to interact with bots through making use of
+outgoing webhook services. It makes it really convenient for users to run multiple bots. Traditionally,
+a person would be required to download zuliprc of each bot and run each of them separately. With Flask
+server, he or she would be able to download a single configuration file for all bots i.e. `flaskbotrc` and
+start flask server which would enable interacting with all the bots.
+
+It's location is `api/flask_bot_server.py`.
+
+### How to run bots using the Flask server
+
+1.  Make sure that bot with whom you want to interact has been setup and is active (lets call it bot1).
+
+2.  Download the `flaskbotrc` through the download icon below `Add a new bot` form. It contains the
+    configuration details for all the active bots. It's structure is very similar to that of zuliprc.
+
+3.  Change directory to `api/` and run the flask server by passing the `flaskbotrc` to it.
+    The command format is:
+    ```
+    cd api
+    python flask_bot_server.py <path_to_flaskbotrc>
+    ```
+    Sample command:
+    ```
+    python flask_bot_server.py ~/flaskbotrc
+    ```
+    Make sure that the flask server is being run inside the vagrant machine.
+
+4.  Now setup the outgoing webhook service which will interact with the flask server. For this create an
+    `outgoing webhook` type bot (lets call this bot2) and create a service with base url of the form:
+    ```
+    http://127.0.0.1:5002/bots/<bot_name>
```

Comment:
Would it clarify things if we explicitly explain this URL to be the vagrant testing server, with other servers (such as `chat.zulip.org` being possible targets as well?